### DISCUSSION TITLE
How to add contact to player phone?

### DISCUSSION THREAD

**Discussion by RazortDev** [scripthookvdotnet/scripthookvdotnet#1319] [Q&A] [answered]
is it possible, we add a contact on the phone, and if we call it brings up UI.Notify?

**Comment by kagikn** [comment]
No plan to directly implement phone UI elements since it may result in SHVDN too bloated. We removed classes like `MenuBase` when we create the first version of v3 API even. You could use a library like [iFruitAddon2](https://github.com/Bob74/iFruitAddon2) (but for v2 scripts) or create your library to add contacts on the phone. The sentence "if we call it brings up UI.Notify" doesn't make sense to me, but anyway I'm not adding phone UI elements.

**Reply by RazortDev** [reply]
i mean, if player call the contact, player will get Notify

**Reply by kagikn** [reply, marked as answer]
I guess you'd like to post a ticker notification on the phone contact event, but I would return the same response, not planning to add/expose events about phones. I'm not experienced with phone stuff either. You'll need to use an external library or write your code on your own to handle phone stuff.
